saw-remote-api/python/saw/llvm.py

```diff
@@ -153,6 +153,24 @@ def to_json(self) -> Any:
         return {'setup value': 'field',
                 'base': self.base.to_json(), 'field': self.field_name}
 
+class GlobalInitializerVal(SetupVal):
+    name : str
+
+    def __init__(self, name : str) -> None:
+        self.name = name
+
+    def to_json(self) -> Any:
+        return {'setup value': 'global initializer', 'name': self.name}
+
+class GlobalLValueVal(SetupVal):
```

Comment:
The name "`GlobalLValueVal`" is, admittedly, somewhat redundant, but I stuck with the `-Val` suffix for consistency with the other `SetupVal` subclasses in this module. If you'd prefer dropping it, I'd be fine with that as well.

Reply:
I like the way you have it -- the consistency is nice.